app/views/jobseekers/profiles/job_preferences/roles.html.slim

```diff
@@ -7,6 +7,14 @@
 
   .govuk-grid-row
     .govuk-grid-column-two-thirds
-      = f.govuk_collection_check_boxes :roles, @step.options, :first, :last, legend: { size: "l" }, caption: { text: "Job preferences", size: "l" }
+      span.govuk-caption-l = "Job preferences"
+      h1.govuk-heading-l class="govuk-!-margin-bottom-2"
+        = "What roles are you interested in?"
+      span.govuk-caption-m class="govuk-!-margin-bottom-8"
+        = "Select all that apply to set up your profile"
```

Comment:
The plain text content could be displayed without the dynamic content operators.
```suggestion
      span.govuk-caption-l Job preferences
      h1.govuk-heading-l class="govuk-!-margin-bottom-2"
        | What roles are you interested in?
      span.govuk-caption-m class="govuk-!-margin-bottom-8"
        | Select all that apply to set up your profile
```